src/site/content/en/lighthouse-performance/lighthouse-max-potential-fid/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Max Potential First Input Delay
+description: |
+  Learn about Lighthouse's Max Potential First Input Delay metric and
+  how to measure and optimize it.
+date: 2019-05-02
+updated: 2019-10-16
+web_lighthouse:
+  - max-potential-fid
+---
+
+Max Potential First Input Delay (FID) is one of the metrics
+tracked in the **Performance** section of the Lighthouse report.
+Each metric captures some aspect of page load speed.
+
+Lighthouse displays Max Potential FID in milliseconds:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="max-potential-fid.png" alt="A screenshot of the Lighthouse Max Potential First Input Delay audit">
+</figure>
+
+## What Max Potential FID measures
+
+Max Potential FID measures the duration of the
+[longest task](/long-tasks-devtools#what-are-long-tasks) after
+[First Contentful Paint][fcp]. It is meant to capture the worst case
+[First Input Delay][fid] that your users might experience. Tasks before
+First Contentful Paint are excluded because it's unlikely that a user will
+attempt to interact with your page before any content has been rendered to the screen,
+which is what First Contentful Paint measures.
+
+## How Lighthouse determines your Max Potential FID score
+
+<!-- TODO(kaycebasques): In the FCP doc we link to the HTTP Archive report of FCP data.
+     If we get a similar report for MPFID we should link to that.
+     https://web.dev/first-contentful-paint/#how-lighthouse-determines-your-fcp-score -->
+
+Your Max Potential FID score is a comparison of your page's Max Potential FID time
+and Max Potential FID times for real websites, based on
+data from the [HTTP Archive](https://httparchive.org).
+For example, if your Max Potential FID score in Lighthouse is green, it means
+that your page performs better than 90% of real websites.
+
+This table shows how to interpret your TBT score:
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Max Potential FID time<br>(in milliseconds)</th>
+        <th>Color-coding</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>0–130</td>
+        <td>Green (fast)</td>
+      </tr>
+      <tr>
+        <td>130-250</td>
+        <td>Orange (average)</td>
+      </tr>
+      <tr>
+        <td>Over 250</td>
+        <td>Red (slow)</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+{% include 'content/lighthouse-performance/scoring.njk' %}
+
+## How to improve your Max Potential FID score
+
+See [How to improve your TTI score][tti]. The strategies for improving Max Potential FID are
```

Comment:
FWIW the strategies to make *iterative* improvements on TTI and mpFID are actually quite different. Because mpFID only cares about the longest task, just yielding in your code/chunking work but not reducing any of it will make quick, big gains for mpFID (and the UX) but do nothing for TTI. This is actually one of my favorite properties of mpFID :D. 

Once you want to make both of them green the strategies forcibly converge :)

Reply:
Good call, thanks for the juicy technical perspective